### PR TITLE
Make !grapple default to current combatant if char not in init

### PR DESCRIPTION
**Pull request by Lathaon**
### What Alias/Snippet is this for?
!grapple

### Summary
Just a one-line change to make !grapple default to current combatant if the active character isn't in init. Makes it more convenient for DMs to use it for monsters.

#### PR Type

- [ ] This PR is a code change that implements a feature request.
- [ ] This PR fixes an issue.
- [x] This PR adds a new feature that is not an open feature request.
- [ ] This PR is not a code change (e.g. documentation, README, ...)
#### Other
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] If code changes were made then they have been tested.
- [ ] I have updated the documentation to reflect the changes.
- [ ] I properly commented my code where appropriate

### CODE REVIEW

File: Collections/Grapple/grapple.alias [hunk 1]
```diff
@@ -3,7 +3,7 @@ embed
 # Baseline Variables
 args     = argparse(&ARGS&)
 c        = combat()
-grappler = args.last("c", name)
+grappler = args.last("c", name if not c or not c.current or c.me else c.current.name)
```
Comment (Croebh) [inline Collections/Grapple/grapple.alias:6]: Should this `not (c or (c.current or c.me))` or something? the `or c.me` is throwing me off

Comment (Croebh) [reply Collections/Grapple/grapple.alias:6]: `not c or not (c.current or c.me)`

Comment (Lathaon) [reply Collections/Grapple/grapple.alias:6]: The `c.me` is to preserve current behaviour if your active character is in combat.

In plain English... Use the active character if the active character is in initiative, or if there's no current combatant. Use current combatant otherwise.

Comment (Lathaon) [reply Collections/Grapple/grapple.alias:6]: Maybe it would be clearer the other way round? `c.current.name if c and c.current and not c.me else name`

Comment (Croebh) [reply Collections/Grapple/grapple.alias:6]: Nah your explanation made sense to me I just was derping haha
